7_react/react1/week1/src/App.js

```diff
@@ -0,0 +1,9 @@
+import './App.css';
+
+function App() {
```

Comment:
It seems like you created the new components, but you are not using it here.
If you want to display the new components you need to import them and place it here in the code:
```
import './App.css';
import TodoList from './TodoList/TodoList';
import Header from './Header/Header';

function App() {
  return (
    <>
      <h1>Srushti Salke</h1>
      <Header />
      <TodoList />
    </>
  );
}

export default App;

```

Reply:
<img width="878" alt="Screenshot 2024-01-14 at 12 30 40" src="https://github.com/HackYourFuture-CPH/class26-homework/assets/15450095/965b17ba-4cb3-4748-8320-0971d58c231c">